assets/css/custom.css

```diff
@@ -0,0 +1,29 @@
+.title-header {padding: .75rem 1.25rem; background-color: #f5f5f5; border-bottom: 1px solid transparent;}
```

Comment:
You should be consistent with your CSS, better have everything not in one line like this but have it with indentation like below.

Reply:
Yes, you got a point. I didn't expend much time review my CSS code.